src/workflows/dags/airqo_measurements.py

```
@@ -489,12 +487,12 @@ def send_hourly_measurements_to_bigquery(airqo_data: pd.DataFrame):
         )
 
     @task(retries=3, retry_delay=timedelta(minutes=5))
-    def send_raw_measurements_to_bigquery(airqo_data: pd.DataFrame):
+    def send_raw_measurements_to_bigquery(data: pd.DataFrame):
         from airqo_etl_utils.airqo_utils import AirQoDataUtils
```

Ruff (0.8.2)

`airqo_etl_utils.airqo_utils.AirQoDataUtils` imported but unused

Remove unused import: `airqo_etl_utils.airqo_utils.AirQoDataUtils`

(F401)